### PR TITLE
Add meta-xt-rcar-domx layer to domd

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: prod-devel-rcar.yaml [hunk 1]
```diff
@@ -65,8 +65,7 @@ common_data:
     - [SOC_FAMILY, "%{SOC_FAMILY}"]
 
     # Add systemd configuration
-    - [DISTRO_FEATURES_append, " systemd"]
```
Comment (rshym) [inline prod-devel-rcar.yaml:68]: This removal may introduce issues. We have few recipes (gles-user-module, optee_client) in meta-xt-rcar and meta-renesas that checks presence of `systemd` in `DISTRO_FEATURES`.
May be it's better to do not remove this line.

Comment (rshym) [reply prod-devel-rcar.yaml:68]: Sorry, my previous comment is not valid. "systemd" is appended in `init-manager-systemd.inc`

**Comment by rshym** [comment]
Acked-by: Ruslan Shymkevych <<ruslan_shymkevych@epam.com>>

**Comment by rshym** [comment]
Have to be merged with https://github.com/xen-troops/meta-xt-rcar/pull/31

**Pull request merged**